doc/specs/stdlib_experimental_error.md

```diff
@@ -77,15 +78,15 @@ program demo_check3
 end program demo_check3
 ```
 
-## `error_stop` - aborts the program
+## `[[stdlib_experimental_error:error_stop]]` - aborts the program
 
 ### Description
 
 Aborts the program with a message and a nonzero exit code.
 
 ### Syntax
 
-`call error_stop(msg, code)`
+`call [[stdlib_experimental_error(module):error_stop(interface)]](msg, code)`
```

Comment:
@jvdp1 I think you were asking about FORD linking syntax. Here ([this file](https://github.com/fortran-lang/stdlib/pull/198/files#diff-5ce6f678ff38888ff6ea95b22117c4e3)) are some examples.

Basically the syntax is put the thing to link to in double square brackets: `[[ ]]`. There is a left hand side and a right hand side separated by a colon (`:`), but only the left hand side is needed. e.g., `[[stdlib_experimental_error]]` and `[[stdlib_experimental_error(module)]]` would both link to the module page: https://stdlib.fortran-lang.org/module/stdlib_experimental_error.html. The parentheses are used to disambiguate the thing in question because you might have multiple entities of the same name in that scope, e.g., a user defined type and a function of the same name to use as a constructor. Unless there is ambiguity the stuff in parenthesis can be omitted. The right hand side (of the colon), if present will provide the text of the link, and link to the anchor for the RHS entity's entry on the LHS entity's page.

Reply:
Thanks @zbeekman for your explanations and example.